COMETwebapp.Tests/ViewModels/Components/BookEditor/BookEditorBodyViewModelTestFixture.cs

```diff
@@ -27,16 +27,19 @@ namespace COMETwebapp.Tests.ViewModels.Components.BookEditor
     using CDP4Common.CommonData;
     using CDP4Common.EngineeringModelData;
     using CDP4Common.ReportingData;
+    using CDP4Common.SiteDirectoryData;
 
     using CDP4Dal;
 
+    using COMET.Web.Common.Enumerations;
     using COMET.Web.Common.Services.SessionManagement;
 
     using COMETwebapp.ViewModels.Components.BookEditor;
 
     using Moq;
 
     using NUnit.Framework;
+    using System.Runtime.Intrinsics.Arm;

```

Comment:
Is this using statement needed?

Reply:
not really xd, removed in the latest commit